dsl/dsl.opam

```diff
@@ -14,6 +14,11 @@ depends: [
   "dune" {>= "3.16"}
   "ounit2"
   "menhir"
+  "core_kernel"
+  "pcap-format"
+  "hex"
+  "csv"
+  "mmap"
```

Comment:
Is is possible to have comments in an .opam file? Maybe have a comment chalking out which things were included for the purposes of `schedulers-in-ocaml`? It'll help us rip it out later if we needed

Reply:
`dsl.opam` is generated by automatically `dune` from `dune-project`; so, it's best not to mess with it. Instead, I've added a comment to `dune-project` with [3a5d1e8](https://github.com/cucapra/packet-scheduling/pull/57/commits/3a5d1e8f830b6a65b267c49447100d6e0e3002f3)